decidim-meetings/lib/decidim/meetings/test/factories.rb

```diff
@@ -262,18 +291,27 @@
   end
 
   factory :meetings_poll_answer, class: "Decidim::Meetings::Answer" do
+    transient do
+      skip_injection { false }
+    end
     questionnaire factory: :meetings_poll_questionnaire
-    question { create(:meetings_poll_question, questionnaire: questionnaire) }
-    user { create(:user, organization: questionnaire.questionnaire_for.organization) }
+    question { create(:meetings_poll_question, questionnaire: questionnaire, skip_injection: skip_injection) }
+    user { create(:user, organization: questionnaire.questionnaire_for.organization, skip_injection: skip_injection) }
   end
 
   factory :meetings_poll_answer_option, class: "Decidim::Meetings::AnswerOption" do
-    question { create(:meetings_poll_question) }
-    body { generate_localized_title }
+    transient do
+      skip_injection { false }
+    end
+    question { create(:meetings_poll_question, skip_injection: skip_injection) }
+    body { generate_localized_title(:meetings_poll_answer_option_body, skip_injection: skip_injection) }
```

Comment:
This is how it's done in the original PR / develop:
```suggestion
    body { generate_localized_title }
```